freqtrade/exchange/hyperliquid.py

```diff
@@ -168,9 +312,18 @@ def get_funding_fees(
         :return: funding fee since open_date
         :raises: ExchangeError if something goes wrong.
         """
+        # Modification for HIP-3 Assets:
+        # Substitute : to - to get funding rates for
+        if ":" in pair.split("/")[0]:
+            base_asset, quote_asset = pair.split("/")
+            base_asset = base_asset.replace(":", "-", 1)
+            pair = f"{base_asset}/{quote_asset}"
+            logger.debug(f"Adjusted HIP-3 pair for funding fee lookup: {pair}")
```

Comment:
Do i understand this correctly that you need to switch from `XYZ:GOOGL/USDT:USDT ` to `XYZ:GOOGL/USDT`  for this to work?
If that's the case - that's a ccxt bug - not something we should aim to unify here.